app/models/payment.rb

```diff
@@ -100,4 +105,12 @@ def successfully_completed
     notify_observers(self)
   end
 
+
+  def note_payor_deleted(payor_type = NOTES_DEFAULT_PAYOR,
+                         payor_email = NOTES_UNKNOWN_EMAIL,
+                         deleted_time = Time.now.utc)
+    deleted_note = "#{payor_type} #{payor_email} for this payment was deleted on #{deleted_time}"
+    update(notes: (notes.nil? ? deleted_note : "#{notes}#{NOTES_SEPARATOR}#{deleted_note}") )
```

Comment:
I suggest using `update_column` here.  If we add `after_update` or `before_update` actions to the Payment model later (we don't have any right now) we probably don't want this notes posting to trigger any such actions (unless we do, in which case we can change the code then).

Reply:
I view it from the perspective of sticking with the convention and expected behavior:  use `update` as you do when you need to update information, let any validations be called automatically (not that there are any for this), and have the `updated_at` column updated automatically.
Later if we need to use callbacks, then it will be clear during testing that this might need to be changed and it can be changed then.

I don't want someone looking at this in the future to have to figure out why this uses `update_column` and manually changes `updated_at` just because of a possibility that it is required in the future.